tasks/pages/ward/[wardId].tsx

```diff
@@ -167,30 +165,30 @@ const WardOverview: NextPage = ({ language }: PropsWithLanguage<WardOverviewTran
       active,
       over
     } = event
-    const overData = over?.data.current
+    const source = active.data.current
+    const target = over?.data.current
+
     const patientId = draggedPatient?.patient?.id ?? draggedPatient?.bed?.patient?.id ?? ''
 
-    if (overData && active.data.current) {
-      if (overData.patientListSection) {
-        // Moving in patientlist
-        if (overData.patientListSection === 'unassigned') {
-          if (active.data.current.discharged) {
+    if (source && target) {
+      if ('patientListSection' in target) {
+        // Moving inside of patient list
+        if (target.patientListSection === 'unassigned') {
+          if ('discharged' in source && source.discharged) {
             readmitPatientMutation.mutate(patientId)
-          } else {
-            unassignMutation.mutate(patientId)
           }
-          unassignMutation.mutate(patientId)
-        } else if (overData.patientListSection === 'discharged') {
+          unassignMutation.mutate(patientId) // TODO: is this correct?, doing two mutations for a patient if previously discharged?
+        } else if (target.patientListSection === 'discharged') { // TODO: can we unify the formats for patients PREVIOUSLY having been discharged and patients BEING discharged? (boolean vs.patientListSection)
```

Comment:
@DasProffi

Reply:
doing two mutations is correct as we need to reactivate a patient, before using it again

a boolean would not be good as we have three possible states active, discharged, assigned. Assigned cannot be dragged to, but this might change later on